### DISCUSSION TITLE
How to get segmentation results in colors image

### DISCUSSION THREAD

**Discussion by DarinKumarnsit** [roboflow/notebooks#142] [Q&A]
Hi, I am playing with [Segment Anything Model](https://github.com/roboflow-ai/notebooks/blob/main/notebooks/how-to-segment-anything-with-sam.ipynb) and only see black and white in segmentation results. So I was wondering if there is any code that can display segmentation results in colors like this? 

![image](https://github.com/roboflow/notebooks/assets/58914019/dd5dee7f-4a69-4c3c-b3a2-c5514e2e5445)

Thank you for your assistance.

**Comment by SkalskiP** [comment]
> and only see black and white in segmentation results

Could you share the code you currently use?

**Reply by DarinKumarnsit** [reply]
Yes, of course.   I use the code from here https://github.com/roboflow-ai/notebooks/blob/main/notebooks/how-to-segment-anything-with-sam.ipynb
